custom_components/tuya_local/devices/drexma_et7aw_thermostat.yaml

```diff
@@ -0,0 +1,324 @@
+name: Drexma WiStat Thermostat
+products:
+  - id: 1oi7spyvsihkwrrr
+    manufacturer: Drexma Industries inc.
+    model: "ET-7AW"
+entities:
+  - entity: climate
+    dps:
+      - id: 101
+        type: boolean
+        name: hvac_mode
+        mapping:
+          - dps_val: false
+            value: "off"
+          - dps_val: true
+            constraint: mode
+            conditions:
+              - dps_val: "Smart"
+                value: auto
+              - dps_val: "Manual"
+                value: heat
+              - dps_val: "Anti_frozen"
+                value: away
```

Comment:
`away` is not a valid hvac_mode.

Probably it is better to just use a simpler mapping for dp 101 where true is always mapped to `heat`, and change mode to `preset_mode`.
If you add `translation_key: thermostat` to the entity definition above (between name and dps), then you can use `manual`, `program` and `away` as values in the mapping to get suitable translated preset names.